odoo/addons/base/models/ir_mail_server.py

```diff
@@ -543,6 +542,19 @@ def _get_default_from_address(self):
         """
         return tools.config.get("email_from")
 
+    @api.model
```

Comment:
in `build_email`, we can also take the context by default (so we are consistent with other methods)

```
email_from = email_from or self._get_default_from_address()

# ->

email_from = email_from or self.env.context.get('notifications_email') or self._get_default_from_address()
```

(I don't think we can simplify, and making `_get_default_from_address` and `_get_default_bounce_address` take from the context because of the overwrite in mail)